src/components/StudyRoom/StudyRoomCreate.js

```diff
@@ -30,11 +31,21 @@ export default function RoomCreation() {
         participants:[],
     });
 
-    React.useEffect(()=>{
-      let user = localStorage.getItem("username")
-        setRoomData({...roomData, owner:user});
-    })
+    React.useEffect(()=> {
+        let user = localStorage.getItem("username")
+        setRoomData({...roomData, owner: user})
+        })
 
+    React.useEffect( ()=>{
+        fetchData();
+    },[])
+    function fetchData() {
+        axios.get('http://localhost:5000/friend/murth')
```

Comment:
url should be / friend / {variable}. You are hard coding the variable to get.

Reply:
not in the scope of this PR